packages/components/pie-button/package.json

```diff
@@ -20,5 +22,10 @@
   "license": "Apache-2.0",
   "volta": {
     "extends": "../../../package.json"
+  },
+  "dependencies": {
+    "@lit-labs/react": "1.1.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
```

Comment:
should these be dependencies, dev dependencies or peer dependencies?

Reply:
I've actually just tested with workspaces and yalc and everything works without these dependencies 😄  Since these dependencies actually exist in the package.json of the react example apps anyway we probably don't need them here too - have removed 👍